### ISSUE TITLE
add blogroll 

### ISSUE THREAD

**Issue by etosch** [CSWomenUMass/cswomenumass.github.io#43]

**Comment by shirki** [comment]
Hey, I do blog about "life hacks" as you called it. Not sure if that's relevant, let me know.

**Comment by etosch** [comment]
Yes, it's relevant if it's related to general issues in research or grad school -- e.g., Lifehacker.com isn't relevant since it's quite general, but blogs focusing on the following topics are: 
-  productivity
- stress management
-  particular life situations    \*  having children while in grad school
  - taking care of aging parents while in grad school    \* managing
    debt in grad school

There were the kinds of topics we had in mind when talking about "life
hacks" or non-technical blog topics appropriate to the blogroll.

## 

Emma Tosch
PhD Candidate, College of Information & Computer Sciences
University of Massachusetts Amherst
etosch@cs.umass.edu http://cs.umass.edu/~etosch

On Sun, Feb 7, 2016, at 01:36 PM, Shiri Dori-Hacohen wrote:

> Hey, I do blog about "life hacks" as you called it. Not sure if that's relevant, let me know.
> 
> —
> Reply to this email directly or view it on GitHub[1]. 

Links:
1. https://github.com/CSWomenUMass/cswomenumass.github.io/issues/43#issuecomment-181075617

**Comment by etosch** [comment]
- http://pgbovine.net/writings.htm, plus add a direct link to PhD Grind memoir (http://pgbovine.net/PhD-memoir/pguo-PhD-grind.pdf)
- http://lindseykuper.livejournal.com/
- http://jxyzabc.blogspot.com/

**Comment by etosch** [comment]
And of course, I have a research blog: http://etosch.github.io

**Comment by etosch** [comment]
http://joelonsoftware.com/
